JDI.Light/JDI.Light/Interfaces/Complex/IRadioButtons.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections.Generic;
+using JDI.Light.Interfaces.Base;
+using OpenQA.Selenium;
+
+namespace JDI.Light.Interfaces.Complex
+{
+    public interface IRadioButtons : IBaseUIElement, IGetValue<string>
+    {
+        By RadioButtonLocator { get; set; }
+        By LabelLocator { get; set; }
+        void Select(string value);
+        void Select(int index);
+        new string Selected();
+        List<string> Values();
+    }
```

Comment:
we can also add something like SelectAll() and UnselectAll()

Reply:
Not sure if this is suitable for radio buttons.

Reply:
Agreed.